Kernel/Devices/Storage/StorageDevice.h

```diff
@@ -58,9 +58,9 @@ class StorageDevice : public BlockDevice {
 
     // ^BlockDevice
     virtual ErrorOr<size_t> read(OpenFileDescription&, u64, UserOrKernelBuffer&, size_t) override;
-    virtual bool can_read(OpenFileDescription const&, u64) const override;
+    virtual bool can_read(OpenFileDescription const&, u64) const override { return true; }
```

Comment:
I think we should return `true` to indicate that we can technically always read from the device, so it shouldn't try to block just because we are past the last block (or offset).